### DISCUSSION TITLE
Correct Way to Publish "Remote" Jobs???

### DISCUSSION THREAD

**Discussion by bscottroberts** [bullhorn/career-portal#393] [Q&A] [answered]
What is the correct way to Publish Remote Jobs in Bullhorn that will show correctly on our Portal site?

I tried the following-
On the Job, set the City to "Remote", set the Country to "None Specified" (so that the State is not required).
Published the job.

Result-
The job shows on our career portal with a Location of "Remote". Step in the right direction.

Issue-
A Candidate can't search for "Remote" in the Location Filter. The field requires a real city/state.

Anyone know the correct way to Publish a "Remote" job??

**Comment by charlesabarnes** [comment, marked as answer]
Hello, do you have a link to your portal?  This doesn't sound like the open source bullhorn career portal because state is not required on it

**Reply by charlesabarnes** [reply]
Heres an example of what this project covers 
http://bh-career-portal-example.herokuapp.com/

**Reply by bscottroberts** [reply]
I think you may be correct... Here is our site- https://jobs.apcinc.com/index.smpl?arg=jb_search_results&view=0

**Reply by bscottroberts** [reply]
We have our site that is not i-framed. I believe this is the Github code.
https://portal.apcinc.com/jobs/index.html#/jobs

I have two jobs that I set with the City as "Remote". One shows in the Location filter (remote, North Caroline). The other that is Remote with no state/country, does not show in the Location list.

**Reply by charlesabarnes** [reply]
Ah @bscottroberts that's the old version of the portal, and city/state was specifically split for this reason.  I would suggest upgrading so you can get that feature.  It also has some nice SEO features that may be beneficial.
